MATURITY_MODEL.md

```diff
@@ -1,5 +1,7 @@
 # QMM, in detail
-Qualitic Maturity Model consists of 5 levels; the first level introduces the initial steps to get in the picture of quality; whereas level 5 indicates the ultimate state of quality.
+Qualitic Maturity Model consists of [5 levels](/LEVELS.html); the first level introduces the initial steps to get in the picture of quality; whereas level 5 indicates the ultimate state of quality.  
```

Comment:
in the pull request #58, ".html" has been removed in levels.html address. does this relate to that or this is my mistake?
By the way, I mean please be careful on this.
Thanks.

Reply:
You're right. I'm gonna fix it.

Reply:
Done.